mod/01_rating/set/abstract/all_values.rb

```diff
@@ -0,0 +1,182 @@
+include_set Type::SearchType
```

Comment:
`all_company_values` sounds confusing to me (it is used to store metric values for companies and metrics). I would agree to rename it to `all_metric_values`